pinot-core/src/main/java/org/apache/pinot/core/routing/TablePartitionInfo.java

```diff
@@ -19,24 +19,26 @@
 package org.apache.pinot.core.routing;
 
 import java.util.List;
-import java.util.Set;
 
 
+/**
+ * Tracks segments by partition for a table. Also tracks the invalid partition segments.
```

Comment:
note: moved existing `TablePartitionInfo` to `TablePartitionReplicatedServersInfo`. This one only tracks segments by partition and the invalid partition segments.